.github/workflows/main.yaml

```diff
@@ -0,0 +1,50 @@
+name: linting and testing
+
+on: [push, pull_request]
+
+jobs:
+  terraform-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+#      - uses: actions/cache@v3
+#        name: Cache plugin dir
+#        with:
+#          path: ~/.tflint.d/plugins
+#          key: ubuntu-latest-tflint-${{ hashFiles('.tflint.hcl') }}
+
+      - uses: terraform-linters/setup-tflint@v3
+        name: Setup TFLint
+        with:
+          tflint_version: v0.44.1
+
+      - name: Show version
+        run: tflint --version
+
+      - name: Init TFLint
+        run: tflint --init
+
+      - name: Run TFLint
+        run: tflint -f compact
+
+  ansible-lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
```

Comment:
Should this be on a newer python release? There's a warning being emitted in the action.

`Warning: ansible-lint is no longer tested under Python 3.8 and will soon require 3.9. Do not report bugs for this version`

Reply:
@hcoyote oh yeah, there's a number of other changes in the pipe too. I probably should have marked this as draft or in progress. since it's an action script i needed a way to test it 😅